PhysicsTools/PatAlgos/python/tools/jetCollectionTools.py

```diff
@@ -3,7 +3,8 @@
 from PhysicsTools.PatAlgos.tools.ConfigToolBase import *
 
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
```

Comment:
From the discussion in this thread, I understand that this import can be removed: please do so, in order to avoid confusion

Reply:
Now removed in the latest commit.